docs/astro/sql/operators/append.rst

```diff
@@ -0,0 +1,43 @@
+======================================
+append operator
+======================================
+
+.. _append_operator:
+
+When to use the ``append`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We can use ``append`` operator when we want to append the source table to the target table.
+
+.. literalinclude:: ../../../../example_dags/example_append.py
+   :language: python
+   :start-after: [START append_example]
+   :end-before: [END append_example]
+
+When used without a columns parameter, AstroSDK assumes that both tables have the same schema.
+
+When tables have same schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. **Case 1:** When complete table needs to be merged. You can skip ``columns`` parameter.
+    .. literalinclude:: ../../../../example_dags/example_append.py
+       :language: python
+       :start-after: [START append_example]
+       :end-before: [END append_example]
```

Comment:
This block is same as the block on line 11-14. Do we need the block on line 11-14 since we're covering it in 2-3 cases, one without columns here and one without columns below.

Reply:
@pankajkoti  It's there to explain the basic and use-case based usage. I think both make sense in their own place. WDYT?

Reply:
No specific opinion as such. Just felt that the code block that would appear will be the same and won't have any difference, so could be a duplicate.

Reply:
@pankajkoti I see your point, I'll look if there is a better way to deal with this.